src/components/Players.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <ul>
+    <li v-for="player in players">
+      {{player.name}}
+    </li>
+  </ul>
+</template>
+
+<script>
+  import { ROUTES } from '../constants'
+  export default {
+    name: 'players',
+    components: {},
+    data () {
+      return {
+        players: []
+      }
+    },
+    created () {
+      this.$http.get(ROUTES.PLAYERS)
```

Comment:
make an external file which wrapp get/then/catch and do not expose $http directly into a file vue